Source/Project64-core/N64System/N64DiskClass.cpp

```diff
@@ -290,7 +290,7 @@ void CN64Disk::ConvertDiskFormat()
         atrack = 0;
         for (track = 0; track < ZoneTracks[zone]; track++)
         {
-            if (track == SystemData[0x20 + zone * 0xC + atrack])
+            if (atrack < 0xC && track == SystemData[0x20 + zone * 0xC + atrack])
```

Comment:
Amount of alternate tracks. Perhaps a constant would be useful but we'll see that later when we'll do development disk support.